zeebe/exporter-test/src/main/java/io/camunda/zeebe/exporter/test/ExporterTestContext.java

```diff
@@ -26,6 +27,11 @@ public final class ExporterTestContext implements Context {
   private Configuration configuration;
   private RecordFilter recordFilter;
 
+  @Override
+  public MeterRegistry getMeterRegistry() {
+    return null;
```

Comment:
:x: Please return a `SimpleMeterRegistry`. It should be a field in the context so it's always the same returned for this particular context, not one we create on the fly here.